spec/models/provider_tag_mapping_spec.rb

```diff
@@ -198,8 +198,8 @@ def map_to_tags(mapper, model_name, labels_kv)
     let(:vm_inventory_object) { amazon_persister.vms.find_or_build("some_ems_ref") }
 
     let(:taggings_collections) do
-      vm_collection       = amazon_persister.inventory_collections[9]
-      taggings_collection = amazon_persister.inventory_collections[8]
```

Comment:
Wow using the array index is not a good idea, going to merge this since it fixes master but the better way to do this is to use the hash key like `persister.vms`